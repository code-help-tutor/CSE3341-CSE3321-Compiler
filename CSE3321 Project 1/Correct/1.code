
123
      and procedure array ARRAY
      (
        ;
        :=

        token
        =
      )
3
while