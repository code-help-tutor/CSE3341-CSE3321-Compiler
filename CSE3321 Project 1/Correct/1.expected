CONST[123]
AND
PROCEDURE
ARRAY
ID[ARRAY]
LPAREN
SEMICOLON
ASSIGN
ID[token]
EQUAL
RPAREN
CONST[3]
WHILE
